gitpod-io/website
# fix: quick fix for the image path

## Description

Fixed the Path of `environment-variables-account-settings.png`

## Related Issue(s)

Fixes # n/a

## How to test

- Go to: https://github.com/gitpod-io/website/blob/main/gitpod/docs/environment-variables.md#using-the-account-settings
- Check, Image Attached is not rendering, because of the wrong path
- It is working now [here](https://github.com/Siddhant-K-code/website/blob/7b63aa179e1ac671700a3f9ecf42ae98b9d0408f/gitpod/docs/environment-variables.md#using-the-account-settings), will fix by this PR.

## Release Notes

```release-note
n/a
```


## Documentation

n/a


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1458"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
